sdk/communication/communication-common/src/credential/url.browser.ts

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+/// <reference lib="dom" />
+
+const url = URL;
+const urlSearchParams = URLSearchParams;
+
+export { url as URL, urlSearchParams as URLSearchParams };
```

Comment:
you should be able to use `URL` without these helper files since it is supported by our min node runtime

Reply:
@xirzec well spotted, thanks! it's fixed now :)